politeiawww/quiesce.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"encoding/hex"
+	"net/http"
+
+	pd "github.com/decred/politeia/politeiad/api/v1"
+	www2 "github.com/decred/politeia/politeiawww/api/www/v2"
+	"github.com/decred/politeia/util"
+)
+
+func (p *politeiawww) toggleQuiesce() (*www2.QuiesceReply, error) {
+	p.bbMtx.Lock()
+	defer p.bbMtx.Unlock()
+	p.quiesce = !p.quiesce
+
+	// Toggle user db quiesce mode
+	p.db.Quiesce()
+	// Setup politeiad /quiesce request
+	challenge, err := util.Random(pd.ChallengeSize)
+	if err != nil {
+		return nil, err
+	}
+	e := pd.Quiesce{
+		Challenge: hex.EncodeToString(challenge),
+	}
+
+	// Send politeiad request
+	_, err = p.makeRequest(http.MethodGet, pd.QuiesceRoute, e)
```

Comment:
What happens if this politeiad call fails?  You'll exit with politeiawww and the userdb quiesced but politeiad not quiesced.  Then you're in a wacky state that you can't get out of with restarting everything.

Reply:
Good point